ocaml/testsuite/tests/language-extensions/language_extensions.reference

```diff
@@ -61,36 +61,60 @@ Succeeded
 
-# can call [set ~enabled:false] when extensions are disallowed [comprehensions disabled]:
+# can call [set ~enabled:false] when in universe no_extensions [comprehensions disabled]:
 Succeeded at redundantly calling [set ~enabled:false]
 
-# can't call [enable] when extensions are disallowed [comprehensions disabled]:
-Failed as expected: Cannot enable extension comprehensions: incompatible with flag -disable-all-extensions
+# can't call [enable] when in universe no_extensions [comprehensions disabled]:
+Failed as expected: Cannot enable extension comprehensions: incompatible with flag -universe no_extensions
 
-# can call [disable] when extensions are disallowed [comprehensions disabled]:
+# can call [disable] when in universe no_extensions [comprehensions disabled]:
 Succeeded at redundantly calling [disable]
 
-# can't call [with_set ~enabled:true] when extensions are disallowed [comprehensions disabled]:
-Failed as expected: Cannot enable extension comprehensions: incompatible with flag -disable-all-extensions
+# can't call [with_set ~enabled:true] when in universe no_extensions [comprehensions disabled]:
+Failed as expected: Cannot enable extension comprehensions: incompatible with flag -universe no_extensions
 
-# can call [with_set ~enabled:false] when extensions are disallowed [comprehensions disabled]:
+# can call [with_set ~enabled:false] when in universe no_extensions [comprehensions disabled]:
 Succeeded at redundantly calling [with_set ~enabled:false]
 
-# can't call [with_enabled] when extensions are disallowed [comprehensions disabled]:
-Failed as expected: Cannot enable extension comprehensions: incompatible with flag -disable-all-extensions
+# can't call [with_enabled] when in universe no_extensions [comprehensions disabled]:
+Failed as expected: Cannot enable extension comprehensions: incompatible with flag -universe no_extensions
 
-# can call [with_disabled] when extensions are disallowed [comprehensions disabled]:
+# can call [with_disabled] when in universe no_extensions [comprehensions disabled]:
 Succeeded at redundantly calling [with_disabled]
 
 # [is_enabled] returns [false] when extensions are disallowed [comprehensions disabled]:
 "comprehensions" is correctly disabled
 
+# can set [Stable] while extensions are disabled [comprehensions disabled]:
+Succeeded at setting universe stable
+
+# can call [Language_extension.(enable Layouts Stable)] when in universe stable [comprehensions disabled]:
+Succeeded at redundantly calling [Language_extension.(enable Layouts Stable)]
+
+# can't call [Language_extension.(enable Comprehensions) ] when in universe stable [comprehensions disabled]:
+Failed as expected: Cannot enable extension comprehensions: incompatible with flag -universe stable
+
+# can't call [Language_extension.(enable Layouts Alpha)] when in universe stable [comprehensions disabled]:
+Failed as expected: Cannot enable extension layouts_alpha: incompatible with flag -universe stable
```

Comment:
Right, this is the same behavior as `-disallow-all-extensions` or `-only-erasable-extensions` previously. Should I add a test for that? I'm not sure it's worth doing since we're going to remove the `-extension` flag later anyway.